### ISSUE TITLE
Modify the interface import mode

### ISSUE THREAD

**Issue by chenc041** [arco-design/arco-design#300]
- [x] I'm sure this does not appear in [the issue list of the repository](https://github.com/arco-design/arco-design/issues)

## Basic Info 
debug source code in local with vite, throw error 
## What are the similar cases of this feature
 no

## What problem does this feature solve?
When i debug the source code in local server with vite, it throw an error named
```
Uncaught SyntaxError: The requested module '/fork-project/arco-design/components/Trigger/interface.ts' does not provide an export named 'TriggerProps'
```
I looked for the cause of this error and found that the cause was the import interface problem
and resolve method is modify interface mode
before 
```tsx
import  { TriggerProps, MouseLocationType } from './interface';
```
after
```tsx
import type { TriggerProps, MouseLocationType } from './interface';
```
this way can resolve the error

So  i hope you can change the import mode at future

## Extra info
reference 
 - [ts-release-note-3.8](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html)
 - [TypeScript type exports fail when type is imported from a module](https://github.com/vitejs/vite/issues/731)

**Comment by PengJiyuan** [comment]
I don't know what should I do in component interface...

**Comment by chenc041** [comment]
> I don't know what should I do in component interface...

I will submit a PR later 🤝

**Comment by chenc041** [comment]
Because  I  try to debug the source code locally, if follow the contributing doc, every change code must should to run yarn build and it is very slow, so I create vite project and use the components code as project dependence and found this error
so I submit this pr.
lastly, do you understand what I said?

**Comment by PengJiyuan** [comment]
> Because I try to debug the source code locally, if follow the contributing doc, every change code must should to run yarn build and it is very slow, so I create vite project and use the components code as project dependence and found this error so I submit this pr. lastly, do you understand what I said?

understand, maybe you can `npm run dev` and `npm link` to debug in vite project

**Comment by chenc041** [comment]
> > Because I try to debug the source code locally, if follow the contributing doc, every change code must should to run yarn build and it is very slow, so I create vite project and use the components code as project dependence and found this error so I submit this pr. lastly, do you understand what I said?
> 
> understand, maybe you can `npm run dev` and `npm link` to debug in vite project

Yes, but If use components code as vite dependent, the speed is fast, I have tested it;

**Comment by chenc041** [comment]
> > Because I try to debug the source code locally, if follow the contributing doc, every change code must should to run yarn build and it is very slow, so I create vite project and use the components code as project dependence and found this error so I submit this pr. lastly, do you understand what I said?
> 
> understand, maybe you can `npm run dev` and `npm link` to debug in vite project

Maybe u can change the bundle tool to esbuild ?
now uses tsc?

**Comment by PengJiyuan** [comment]
> > > Because I try to debug the source code locally, if follow the contributing doc, every change code must should to run yarn build and it is very slow, so I create vite project and use the components code as project dependence and found this error so I submit this pr. lastly, do you understand what I said?
> > 
> > 
> > understand, maybe you can `npm run dev` and `npm link` to debug in vite project
> 
> Maybe u can change the bundle tool to esbuild ? now uses tsc?

OK, I will try when I have time, or pr welcome

**Comment by chenc041** [comment]
> > > > Because I try to debug the source code locally, if follow the contributing doc, every change code must should to run yarn build and it is very slow, so I create vite project and use the components code as project dependence and found this error so I submit this pr. lastly, do you understand what I said?
> > > 
> > > 
> > > understand, maybe you can `npm run dev` and `npm link` to debug in vite project
> > 
> > 
> > Maybe u can change the bundle tool to esbuild ? now uses tsc?
> 
> OK, I will try when I have time, or pr welcome

ok
